### PR TITLE
Renames 'noBabel' config option to a more descriptive 'babelIgnore'

### PR DESCRIPTION
**Pull request by rudeworks**
```noBabel``` sounds like an option requiring a boolean, I'm proposing ```babelIgnore``` as a more descriptive option name.

### CODE REVIEW

File: etc/webpack.base.js [hunk 1]
```diff
@@ -16,7 +16,8 @@ module.exports = new WebpackConfig().merge({
   module: {
     loaders: [{
       test: /\.js$/,
-      loader: 'babel', exclude: config.noBabel
+      loader: 'babel',
+      exclude: config.BabelIgnore
```
Comment (dmbch) [inline etc/webpack.base.js]: I think it ought to be `config.babelIgnore` ;)

File: lib/render.js [hunk 2]
```diff
@@ -2,7 +2,7 @@
 var config = require('../lib/config');
 
 require('babel-register')({
-  ignore: config.noBabel
+  ignore: config.BabelIgnore
```
Comment (dmbch) [inline lib/render.js]: See above.

**Comment by dmbch** [comment]
Thanks! This is way better semantically.

File: etc/webpack.base.js (hunk 1 above)
Comment (rudeworks) [inline etc/webpack.base.js]: fixed 🐕💨

**Pull request merged**